### ISSUE TITLE
Logout (delete authorization)

### ISSUE THREAD

**Issue by michael** [michael/hub#11]
I'd like to implement a proper logout. However I'd need an API endpoint to destroy an existing authorization. Probably by just sending the token that should be invalidated?

DELETE /api/v1/authorizations/TOKEN

**Comment by yuchi** [comment]
I don't know. Should the application logout simply by forgetting the access_token?

**Comment by michael** [comment]
Yeah that be a way. Just thought it might be more secure to safely logout and invalidate a token. (in case someone sniffed it etc.)

Not high priority though.

**Comment by yuchi** [comment]
(See how GitHub manage authorizations here: https://github.com/settings/applications )

**Comment by michael** [comment]
I see. Well, let's make this a low priority. At some point we'll add login/admin panel at substance.io anyway.
